book/src/advanced-release-candidates.md

```diff
@@ -15,7 +15,7 @@ ergonomics of new features or changes.
 Github will clearly show such releases as a "Pre-release" and they *will not* show up on
 [sigp/lighthouse/releases/latest]. However, release candidates *will* show up on the
 [sigp/lighthouse/releases] page, so **please pay attention to avoid the release candidates when
-you're looking for stable Lighthouse**.
+you're looking for a stable Lighthouse**.
```

Comment:
```suggestion
you're looking for stable Lighthouse**.
```

Stable here refers to the `stable` branch, perhaps we could say "the stable version of Lighthouse".